hieradata/dev/wso2/common.yaml

```diff
@@ -55,6 +55,9 @@ wso2::template_list:
 #wso2::file_list:
 #  - repository/conf/security/cipher-tool.properties
 #  - repository/conf/security/cipher-text.properties
+#  - repository/conf/security/secret-conf.properties
+
+#wso2::file_list_copy_without_refresh:
```

Comment:
It might be better to rename this variable name. Shall we call it "service_refresh_file_list" and add a comment explaining its behaviour?